processor/PlanExecutor_test.go

```diff
@@ -50,6 +51,12 @@ var _ = Describe("Plan Executor", func() {
 		configuration.FilePath = file.Name()
 		bar = NullProgressBar{}
 		aggregator = statistics.NewAggregator(metrics.DefaultRegistry)
+		channel = make(chan core.ExecutionResult)
+
+		go func() {
+			for range channel {
+			}
```

Comment:
Yeh - only so that I can write to the channel and not have to know how many messages I am going to send.  I could avoid this loop and instead create a buffered channel i.e.

```
var channel = make(channel, numberOfMessages)
```

To avoid having to think of what value numberOfMessages is going to be I simply did that iteration so that the channels would drain but I am not interested in the message that is on the channel.